packages/next/src/build/index.ts

```diff
@@ -1030,7 +1030,7 @@ export default async function build(
         const turboJson = findUp.sync('turbo.json', { cwd: dir })
         // eslint-disable-next-line no-shadow
         const packagePath = findUp.sync('package.json', { cwd: dir })
-        let binding = await loadBindings()
+        let binding = await loadBindings(config.experimental.useWasmBinary)
```

Comment:
To make it the same as jest.ts below:
```suggestion
        let binding = await loadBindings(config?.experimental?.useWasmBinary)
```